test/simpleLP.jl

```diff
@@ -1,40 +1,142 @@
+push!(LOAD_PATH, "/home/guilhermebodin/Documents/Github/Dualization.jl/src")
+using Pkg
+Pkg.activate(".") # Just to make sure to use JuMP 0.19.1
+using MathOptInterface, JuMP, Dualization
+
+const MOI  = MathOptInterface
+const MOIT = MathOptInterface.Test
+const MOIU = MathOptInterface.Utilities
+const MOIB = MathOptInterface.Bridges
+
+MOIU.@model(Model,
+            (MOI.ZeroOne, MOI.Integer),
+            (MOI.EqualTo, MOI.GreaterThan, MOI.LessThan, MOI.Interval,
+             MOI.Semicontinuous, MOI.Semiinteger),
+            (MOI.Reals, MOI.Zeros, MOI.Nonnegatives, MOI.Nonpositives,
+             MOI.SecondOrderCone, MOI.RotatedSecondOrderCone,
+             MOI.GeometricMeanCone, MOI.ExponentialCone, MOI.DualExponentialCone,
+             MOI.PositiveSemidefiniteConeTriangle, MOI.PositiveSemidefiniteConeSquare,
+             MOI.RootDetConeTriangle, MOI.RootDetConeSquare, MOI.LogDetConeTriangle,
+             MOI.LogDetConeSquare),
+            (MOI.PowerCone, MOI.DualPowerCone, MOI.SOS1, MOI.SOS2),
+            (MOI.SingleVariable,),
+            (MOI.ScalarAffineFunction, MOI.ScalarQuadraticFunction),
+            (MOI.VectorOfVariables,),
+            (MOI.VectorAffineFunction, MOI.VectorQuadraticFunction))
+
 #= 
 min -4x1 -3x2 -1
   s.a.
     2x1 + x2 + 1 <= 4
-    x1 + 2x2 + 1 <= 4
+    x1 + 2x2 <= 4
     x1 >= 1
     x2 >= 0
 =#
+function create_primal()
+    primal_model = Model{Float64}()
+
+    X = MOI.add_variables(primal_model, 2)
+
+    MOI.add_constraint(primal_model, 
+        MOI.ScalarAffineFunction(
+            [MOI.ScalarAffineTerm(2.0, X[1]), MOI.ScalarAffineTerm(1.0, X[2])], 1.0),
+            MOI.LessThan(4.0))
+
+    MOI.add_constraint(primal_model, 
+        MOI.ScalarAffineFunction(
+            [MOI.ScalarAffineTerm(1.0, X[1]), MOI.ScalarAffineTerm(2.0, X[2])], 1.0),
+            MOI.LessThan(4.0))
+
+    MOI.add_constraint(primal_model, 
+        MOI.SingleVariable(X[1]),
+            MOI.GreaterThan(1.0))
+
+    MOI.add_constraint(primal_model, 
+        MOI.SingleVariable(X[1]),
+            MOI.GreaterThan(3.0))
+
+    MOI.set(primal_model, 
+        MOI.ObjectiveFunction{MOI.ScalarAffineFunction{Float64}}(), 
+        MOI.ScalarAffineFunction(MOI.ScalarAffineTerm.([-4.0], [X[2]]), -1.0)
+        )
+
+    MOI.set(primal_model, MOI.ObjectiveSense(), MOI.MIN_SENSE)
+    return primal_model
+end
+
+@benchmark primal_model = create_primal()
+@benchmark dualmodel = dualize(primal_model)
+
+using Clp
```

Comment:
we could use get_function and get set to check if the model is the expected one...